ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh

```diff
@@ -0,0 +1,222 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+CLUSTER_NAME=${CLUSTER_NAME:-$NAMESPACE}
+ACCOUNT_ROLES_PREFIX=${ACCOUNT_ROLES_PREFIX:-$NAMESPACE}
+COMPUTE_MACHINE_TYPE=${COMPUTE_MACHINE_TYPE:-"m5.xlarge"}
+OPENSHIFT_VERSION=${OPENSHIFT_VERSION:-}
+MULTI_AZ=${MULTI_AZ:-false}
+ENABLE_AUTOSCALING=${ENABLE_AUTOSCALING:-false}
+ETCD_ENCRYPTION=${ETCD_ENCRYPTION:-false}
+DISABLE_WORKLOAD_MONITORING=${DISABLE_WORKLOAD_MONITORING:-false}
```

Comment:
@yasun1 for those parameters with non-default values, I suggest to reflects them in the workflow name, e.g.  `ENABLE_AUTOSCALING`, `ETCD_ENCRYPTION`, so everyone know the cluster configurations while checking the logs, WDYT?

Reply:
It is a balance between the flexiable and the fixed. Reflecting too many parameters in the workflow name makes the directory structure complex. Such `ENABLE_AUTOSCALING` is an option of `rosa create cluster`,  we have many options for this command, and it is impossible for us to organize the workflows by the options. For the workflow, actually, the user first just needs to know it is a rosa sts cluster with the nightly version. If the user wants to know more, it can use `rosa describe cluster -c <cluster name> -o json` to get a more detailed cluster configuration rather than checking the logs at any time.

Reply:
@yasun1 thanks for detailed explanation, it makes sense to me, users can reflect them in different workflows' name if they want.